tasks/java.yml

```diff
@@ -2,6 +2,8 @@
 - name: "Check if Java is installed"
   shell: which java
```

Comment:
How do you feel about using `command -v java` rather than `which`?

See: http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script